server/expression/executor.go

```diff
@@ -60,10 +60,6 @@ func (e Executor) Statement(statement string) (string, string, error) {
 	}
 
 	err = compare(parsedStatement.Comparator, leftValue, rightValue)
-	if err != nil {
-		return leftValue.String(), rightValue.String(), err
-	}
-
```

Comment:
If you look at the next lines, you will see that this `err` will be returned after formatting the duration fields. So it's safe to remove this condition.